### ISSUE TITLE
ENH: quantum optics book 

### ISSUE THREAD

**Issue by nathanshammah** [qutip/qutip#1116]
The recently introduced [`jupyter-book`](https://github.com/jupyter/jupyter-book) feature allows to collate jupyter notebooks and customize the html layout to form a hypertext that can be used as a book. 

QuTiP already contains quite a lot of lecture material: J.R. Johansson quantum mechanics lectures, as well as other notebooks and examples that are hosted at http://qutip.org/tutorials.

This endeavor could be minimal (just collating notebooks) or more extensive, including harmonizing the content. I could be coordinated by one or a few editors and  possibly several contributors. 

Having something definite as a "book", may induce a reader to "pick it up" more easily.

**Comment by amcdawes** [comment]
I just stumbled across this issue in updating some notebooks and would invite collaboration on this (struggling) effort: https://github.com/amcdawes/qutip-book which was born out of the "lab" notebooks I developed for teaching undergraduate quantum mechanics. The concept you describe here, bringing the existing notebooks together, could be merged or developed into vol 1 and vol 2 with intro to QM vs "advanced" ideas.

**Comment by hodgestar** [comment]
@amcdawes It would be great to organize a more collaborative team. Making nice books is a lot of work, so it would be good to be able to share the work. Is there a placing you're collaborating or discussing already? If we can find an enthusiastic group of people, perhaps we could "bless" this effort by bringing it more officially into the QuTiP fold (e.g. publishing the HTML version of the book on the QuTiP website, bring the repository into the qutip GitHub organization -- whatever seems useful).

**Comment by amcdawes** [comment]
I agree, and at this point, I've only been able to port my main notebooks
over to the book format, but there is certainly a lot of room for other
content (especially advanced topics beyond the undergrad level). It is also
not particularly self-contained as I use it as a supplement to a textbook
for my class.

So far, there have been a few interested participants, but no the project
is not active by any stretch of the imagination.

The visibility QuTiP could bring would certainly help move things forward.
Happy to help make that happen however folks would like to do it!

**Comment by hodgestar** [comment]
@amcdawes I am just wondering how your book is doing?
